packages/calypso-products/src/is-big-sky-onboarding.ts

```diff
@@ -0,0 +1,26 @@
+import { getPlansListExperiment } from './experiments';
+
+declare global {
+	interface Window {
+		__a8cBigSkyOnboarding?: boolean;
+	}
+}
+
+export function isBigSkyOnboarding(): boolean {
+	if ( new URLSearchParams( location.search ).has( 'big-sky-checkout' ) ) {
+		return true;
+	}
```

Comment:
@vykes-mac I added a query parameter for the checkout screen case, so that the message only appears when the user actually chooses Big Sky